Onboarding post (#31)
seedcase-project/team: entries/onboarding.md

---
title: Onboarding
date: last-modified
---

{{< include /includes/_wip.qmd >}}

## Welcome to Seedcase

> If you landed on this post, you are most likely a new team member on the Seedcase project. Welcome!

Reviewer: ```suggestion
If you landed on this post, you are most likely a new team member on the Seedcase Project. Welcome!
```

> To get you started, there are some first steps you'll need to do and some posts to for you to read and review.

Reviewer: ```suggestion
To get you started, there are some first steps you'll need to do and some posts for you to read and review.
```

## First steps

1. **Send your GitHub username and Discord username to the Team Leader**

>     - We use Discord as our communication platform and this way so you can be added to the Seedcase Project server asap. If you don't have a Discord account, you can go to [discord.com](https://discord.com) and create one.

Reviewer: ```suggestion
    - We use Discord as our communication platform and this way so you can be added to the Seedcase Project server right away. If you don't have a Discord account, you can go to [discord.com](https://discord.com) and create one.
```

    - Any subpages of particular interest? Maybe "Welcome and getting started (as a PhD student?)", "General practicalities" and "IT basics"?  
2. [Our Seedcase Team Docs website](https://team.seedcase-project.org/) (which is where this post is located)
    - This website covers the documentation, contributing guidelines, onboarding material, and event details/agendas for the Seedcase Project team
3. [General contributing guidelines website](https://seedcase-project.org/community/contributing)
    - This website covers the contributing guidelines of the Seedcase project
    - Potentially start with the "Decisions" section, describing *why* we choose to use the tools that we use: [Why Discord?](https://seedcase-project.org/community/decisions/why-discord/) and [Why Quarto?](https://seedcase-project.org/community/decisions/why-quarto/)
    - Then go to the [individual guide entries](https://seedcase-project.org/community/contributing) covering guidelines for contributing
4. [Design documentation](https://seedcase-project.org/design/)
    - The design documentation covers stakeholders, design decisions, software architecture, data architecture, and more.

> If something is unclear to you, you are more than welcome to reach out to the other team members on Discord. We'll love to help you out.

Reviewer: ```suggestion
If something is unclear to you, you are more than welcome to reach out to the other team members on Discord. We'd love to help you out.
```
